disco_aws_automation/disco_cloudfront.py

```diff
@@ -0,0 +1,194 @@
+                    'Quantity': 1,
+                    'Items': [
+                        {
+                            'PathPattern': '*',
+                            'TargetOriginId': full_origin_path,
+                            'ForwardedValues': {
+                                'QueryString': True,
+                                'Cookies': {
+                                    'Forward': 'none',
+                                    'WhitelistedNames': {
+                                        'Quantity': 0,
+                                    }
+                                },
+                                'Headers': {
+                                    'Quantity': 0,
+                                }
+                            },
+                            'TrustedSigners': {
+                                'Enabled': False,
+                                'Quantity': 0,
+                            },
+                            'ViewerProtocolPolicy': 'allow-all',
+                            'MinTTL': 86400,
+                            'AllowedMethods': {
+                                'Quantity': 2,
+                                'Items': [
+                                    'HEAD',
+                                    'GET',
+                                ],
+                                'CachedMethods': {
+                                    'Quantity': 2,
+                                    'Items': [
+                                        'HEAD',
+                                        'GET',
+                                    ]
+                                }
+                            },
+                            'SmoothStreaming': False,
+                            'DefaultTTL': 86400,
+                            'MaxTTL': 86400,
+                            'Compress': True
+                        },
+                    ]
+                },
+                'CustomErrorResponses': {
+                    'Quantity': 1,
+                    'Items': [
+                        {
+                            'ErrorCode': 404,
+                            'ResponsePagePath': '/Error.html',
+                            'ResponseCode': '200',
+                            'ErrorCachingMinTTL': 86400
+                        },
+                    ]
+                },
+                'Comment': 'Cloudfront Distribution',
+                'Logging': {
+                    'Enabled': True,
+                    'IncludeCookies': True,
+                    'Bucket': self._get_option("log_bucket_name"),
+                    'Prefix': 'log'
+                },
+                'PriceClass': 'PriceClass_100',
+                'Enabled': True,
+                'ViewerCertificate': {
+                    'CloudFrontDefaultCertificate': True,
+                    'MinimumProtocolVersion': 'TLSv1',
+                },
+            })
+        print response
```

Comment:
May want to indicate what is being printed or remove if this was simply for development.